src/config.js

```diff
@@ -1223,7 +1284,15 @@ class Config {
   }
 
   emitChangeEvent () {
-    if (this.transactDepth <= 0) { return this.emitter.emit('did-change') }
+    if (this.transactDepth <= 0) {
+      return this.emitter.emit('did-change') && this.emitter.emit('did-update')
+    }
+  }
+
+  emitUpdateEvent (keypath='', newValue={}) {
```

Comment:
The `=` signs should have spaces before and after them.
Before:
`emitUpdateEvent (keypath='', newValue={}) {`
After:
`emitUpdateEvent (keypath = '', newValue = {}) {`